views/Home.jsx

```diff
@@ -1,11 +1,23 @@
 const React = require('react');
+const Layout = require('./Layout');
 
 function Home() {
 
   return (
-    <main>
-      <h1>Home</h1>
-    </main>
+    <Layout>
+      <main>
+        <h1>Home</h1>
+        <div>
+          <img src="../public/images/beer.png" alt="Beer-pic"/>
```

Comment:
The image is not correctly linked so it's not displaying. You should put:

`<img src="../images/beer.png" alt="Beer-pic"/>`